libs/components/ag-grid/src/lib/modules/ag-grid/types/lookup-properties.ts

```diff
@@ -9,27 +9,106 @@ import {
+   */
   ariaLabelledBy?: string;
-  /** @deprecated */
+  /**
+   * The value to provide to the autocomplete attribute on the form input.
+   * @default "off"
+   * @deprecated
+   */
   autocompleteAttribute?: string;
+  /**
+   * The data source for the lookup cell to search when users enter text. You can specify static data, such as an array of objects, or you can pull data from a database.
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide data to the lookup component.
+   */
   data?: unknown[];
+  /**
+   * How many milliseconds to wait before searching while users enter text in the lookup field.
+   * @default 0
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Whether to disable the lookup cell.
+   * @default false
+   */
   disabled?: boolean;
+  /**
+   * Whether to enable users to open a picker where they can view all options.
+   * @default false
+   */
   enableShowMore?: boolean;
+  /**
+   * The object property that represents the object's unique identifier. Specifying this property enables token animations and more efficient rendering. This property is required when using `enableShowMore` and `searchAsync` together.
+   */
   idProperty?: string;
+  /**
+   * Placeholder text to display in the lookup field.
+   */
   placeholderText?: string;
+  /**
+   * The array of object properties to search when using the `data` property and the built-in search function.
+   * @default ["name"]
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide data to the lookup component.
+   */
   propertiesToSearch?: string[];
+  /**
+   * The function that dynamically manage the data source when users change the text in the lookup field. The search function must return an array or a promise of an array.
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide searched data to the lookup component.
+   */
   search?: SkyAutocompleteSearchFunction;
+  /**
+   * Fires when users enter new search information and allows results to be returned via an observable. The event is also fired when the "Show more" picker is opened without search text.
+   */
   searchAsync?: (args: SkyAutocompleteSearchAsyncArgs) => void;
+  /**
+   * The array of functions to call against each search result. This filters the search results when using the data input and the default search function. When the `search` property specifies a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```

Comment:
```suggestion
   * The array of functions to call against each search result. This filters the search results when using the `data` input and the default search function. When the `search` property specifies a custom search function, you must manually apply filters inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
```
Sorry I missed this yesterday, but I think "data" should have tickmarks since it's an input name.